cyences_app_for_splunk/default/data/ui/views/cs_windows_reports.xml

```diff
@@ -0,0 +1,129 @@
+<form theme="dark" version="1.1">
+  <label>Windows</label>
+  <search>
+    <done>
+      <set token="tkn_host_stats">$result.host_name$</set>
+    </done>
+    <query>| makeresults | eval host_name = if("*"=="$tkn_host$","","host")</query>
+    <earliest>-24h@h</earliest>
+    <latest>now</latest>
+  </search>
+  <fieldset submitButton="false"></fieldset>
+  <row>
+    <panel>
+      <html>
+        <p>Windows Reports</p>
+        <p>
+          <b>Data Collection:</b> WinEventLog:Security events from <a href="https://splunkbase.splunk.com/app/742/" target="_blank">Splunk Add-on for Windows</a>.</p>
+      </html>
+      <table>
+        <search>
+          <query>`cs_wineventlog_security` | head 1 | stats count | eval data=if(count&gt;0, "Data Present", "Data Not Present"), label="`cs_wineventlog_security`" | table label, data</query>
+          <earliest>$timeRange.earliest$</earliest>
+          <latest>$timeRange.latest$</latest>
+        </search>
+        <option name="drilldown">none</option>
+        <option name="refresh.display">progressbar</option>
+        <format type="color" field="data">
+          <colorPalette type="map">{"Data Present": #03991a, "Data Not Present": #a3030b}</colorPalette>
+        </format>
+      </table>
+    </panel>
+  </row>
+  <row>
+    <panel>
+      <input type="time" token="timeRange">
+        <label>Time Range</label>
+        <default>
+          <earliest>-24h@h</earliest>
+          <latest>now</latest>
+        </default>
+      </input>
+      <input type="text" token="tkn_host" searchWhenChanged="true">
+        <label>Host</label>
+        <default>*</default>
+        <prefix>*</prefix>
+        <suffix>*</suffix>
+      </input>
+      <input type="text" token="tkn_user" searchWhenChanged="true">
+        <label>User</label>
+        <default>*</default>
+        <prefix>*</prefix>
+        <suffix>*</suffix>
+      </input>
+    </panel>
+  </row>
+  <row depends="$hidden_row$">
```

Comment:
This seems like duplicated. We have same search at line no. 7

Reply:
Removed duplicated search from line no. 7.